Doc/whatsnew/3.7.rst

```diff
@@ -435,6 +435,12 @@ API and Feature Removals
   Python 3.1, and has now been removed.  Use the :func:`~os.path.splitdrive`
   function instead.
 
+* :func:`collections.namedtuple` no longer supports the *verbose* parameter
+  or ``_source`` attribute which showed the generated source code for the
+  named tuple class.  This was part of an optimization designed to speed-up
```

Comment:
I think it is worth to add a note about an optimization in the "Optimizations" section.

Reply:
I don't want to litter the docs with more notes about this.  The more important part to note is the feature removal in whatsnew.  The optimization is mostly negligible and insignificant to most users (normal startup doesn't use named tuples at all; a module using one named tuple was using only 0.3ms which was just under 1% of the overall start-up time; formerly, we could make 3 namedtuple classes per millisecond and now we can make about 15).  It is one of the least important optimizations we could have done.